gl/gloffscreen_egl.c

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
It's been a while, but I believe both work. That's also the reason why the EGL hack is included for apitrace.
Should be tested anyway.

I think the EGL code is horrible though because GLEW didn't support EGL at the time (does it now?). So I had to use the EGL extension mechanism. There should be more changes in the nv2a code to actually use the new "glo_get_extension_proc" (which is unfortunaly not the proper way on macs - not sure if we should use macros to make it all work at once). It's a lot of abuse and hackery.
Also some formatting issues / style issues here and there [my fault].